composer.json

```diff
@@ -48,6 +48,8 @@
 		"phpunit": "@test",
 		"phpcs": "@php ./vendor/bin/phpcs . --basepath=.",
 		"phpcbf": "@php ./vendor/bin/phpcbf . --basepath=.",
+		"phpcs-escape": "@php ./vendor/bin/phpcs src --extensions=php -sp --standard=BuddyPress --sniffs=WordPress.Security.EscapeOutput",
+		"phpcompat": "@php ./vendor/bin/phpcs --standard=PHPCompatibilityWP --extensions=php -sp --runtime-set testVersion 5.6- src tests",
```

Comment:
@imath I think this should be 6.1, right? Instead of 5.6

Reply:
@renatonascalves It's PHP version, and we still support version 5.6 as long as we support WordPress versions that are supporting it. We'll be able to upgrade it to 7.0.0 once we raised our minimal WordPress version to 6.3 😓

Reply:
Ops! That's true. Sorry, my mistake. Got confused.